packages/trader/src/App/Components/Elements/ContractAudit/contract-audit.tsx

```diff
@@ -40,18 +44,25 @@ const ContractAudit = ({
     const getSortedUpdateHistory = (history: TContractUpdateHistory) =>
         history.sort((a, b) => Number(b?.order_date) - Number(a?.order_date));
 
+    const requestUpdatedHistory = React.useCallback((id?: number) => {
+        if (!id) return;
+        WS.contractUpdateHistory(id).then((response: TResponse) => {
+            setUpdateHistory(getSortedUpdateHistory(response.contract_update_history));
+        });
```

Comment:
Do we need a catch block here for error handling

Reply:
good question, it was not there before. we can think of adding one but the question is 'how do we want to handle it?' :)

Reply:
Since it's gonna be a network error if it happens, we don't wanna log it, so that it doesn't end up in TrackJS. Also, we don't have any business requirement on whether or not and how we want to handle it. Hence, i'll put `.catch(() => null)` to silence it.